session.js

```diff
@@ -0,0 +1,52 @@
+import { isValidApplePayGateway } from './validateApplePayGateway';
+import responseJson from './responseJson';
+import request from 'request';
+
+export const create = async (event, context, callback) => {
+
+    const serverlessRequest = JSON.parse(event.body) || {url: '', body: {}};
```

Comment:
this will not get to the OR expression if JSON.parse throws an exception. ie: `JSON.parse('asfdas') || {url: '', body: {}};`

Reply:
fixed